ci/release.sh

```diff
@@ -128,3 +128,4 @@ release "UnrealGDKExampleProject" "$(buildkite-agent meta-data get gdk-source-br
 release "UnrealGDKTestGyms"       "$(buildkite-agent meta-data get gdk-source-branch)" "${GDK_VERSION}-rc" "${DRY_RUN_PREFIX}release" "$(buildkite-agent meta-data get UnrealGDKTestGyms-$(buildkite-agent meta-data get gdk-source-branch)-pr-url)"       "spatialos"
 release "UnrealGDKEngineNetTest"  "$(buildkite-agent meta-data get gdk-source-branch)" "${GDK_VERSION}-rc" "${DRY_RUN_PREFIX}release" "$(buildkite-agent meta-data get UnrealGDKEngineNetTest-$(buildkite-agent meta-data get gdk-source-branch)-pr-url)"  "improbable"
 release "TestGymBuildKite"        "$(buildkite-agent meta-data get gdk-source-branch)" "${GDK_VERSION}-rc" "${GDK_VERSION}"           "$(buildkite-agent meta-data get TestGymBuildKite-$(buildkite-agent meta-data get gdk-source-branch)-pr-url)"        "improbable"
+release "UnrealGDKBuild"          "$(buildkite-agent meta-data get gdk-source-branch)" "${GDK_VERSION}-rc" "${GDK_VERSION}"           "$(buildkite-agent meta-data get UnrealGDKBuild-$(buildkite-agent meta-data get gdk-source-branch)-pr-url)"          "improbable"
```

Comment:
do we need a release branch? I assume not

Reply:
Hm, I think it could be useful. Also, it's probably easier to just do it, I worry a bit about what the release pipeline will do if the release branch does not actually exist at the end.

Reply:
So. We keep this the same yeah?